module3/homework/advanced_calculator/advancedCalculator.cpp

```diff
@@ -0,0 +1,114 @@
+#include "advancedCalculator.hpp"
+
+#include <algorithm>
+#include <array>
+#include <cmath>
+#include <functional>
+#include <map>
+#include <regex>
+#include <string_view>
+
+constexpr std::string_view ERROR_CODE_MESSAGE[] = {"Result: ",
```

Comment:
Nice usage of std::string_view.

Please read this, because sometimes is not the best option, and const char* will be better.
https://stackoverflow.com/questions/58281928/any-gotchas-replacing-global-const-char-with-constexpr-string-view

Reply:
Oki I will read this ;~}